dev/CommonStyles/Button_themeresources_v1.xaml

```diff
@@ -56,16 +56,16 @@
             <StaticResource x:Key="AccentButtonBorderBrushPressed" ResourceKey="SystemControlHighlightTransparentBrush" />
             <StaticResource x:Key="AccentButtonBorderBrushDisabled" ResourceKey="SystemControlDisabledTransparentBrush" />
             <StaticResource x:Key="ButtonBackground" ResourceKey="SystemControlBackgroundBaseLowBrush" />
-            <StaticResource x:Key="ButtonBackgroundPointerOver" ResourceKey="SystemControlBackgroundBaseLowBrush" />
-            <StaticResource x:Key="ButtonBackgroundPressed" ResourceKey="SystemControlBackgroundBaseMediumLowBrush" />
+            <StaticResource x:Key="ButtonBackgroundPointerOver" ResourceKey="SystemColorHighlightTextColor" />
```

Comment:
@ranjeshj do we want to update the v1 resources or leave those files untouched?

Reply:
I updated those files since you said in the issues that those updates are based on potential problems, not pure "design" updates and as such shouldn't depend on old vs new styles.

Reply:
@YuliKl Is this currently an issue with the v1 styles? If it is an improvement, then it makes sense, but @chingucoding we would have to test the v1 styles and verify as well.

Reply:
V1 output also looked fine @ranjeshj.